pages/Documents/Reporting/OutboundReportingAPI/overview.md

```diff
@@ -166,17 +167,26 @@ Click [**Account**](https://proactive-messaging.z1.fs.liveperson.com/api/api-doc
 
 ```json
 {
- "filters": {
-   "channels": [
-     "sms"
-   ],
-   "skills": [
-     "billing"
-   ]
- }
+ "channels": [
+   "sms"
+ ],
+ "skills": [
+   "billing"
+ ],
+ "messagestatus": [
```

Comment:
@jschatzman-lp We dont have messagestatus in this account analytics api. Please can you recheck and update this